packages/hagrid/hagrid/cli.py

```diff
@@ -2532,9 +2550,22 @@ def quickstart_cli(
             )
 
         if url:
-            file_path, _ = quickstart_download_notebook(
-                url=url, directory=directory, reset=reset
-            )
+            url_is_valid = validators.url(url)
```

Comment:
We don't want to make hagrid really slow to install by adding lots of unrequired dependencies. You can do this with urllib which is built into python.
```
from urllib.parse import urlparse
```

Reply:
Done via https://github.com/OpenMined/PySyft/pull/6686/commits/e055b18f8a2e9733ee3c014e436ac8073e6d6ed7